src/components/dataset/overview/connection-to-forms.vue

```diff
@@ -24,6 +24,10 @@ except according to the terms contained in the LICENSE file.
               <form-link :form="form"
                 :to="publishedFormPath(form.projectId, form.xmlFormId)"
                 v-tooltip.text/>
+              <span v-if="form.repeatPath" class="repeat-tag"
+                v-tooltip.aria-describedby="form.repeatPath">
```

Comment:
Can use `v-tooltip.no-aria` to explicitly not be read by screenreader (could be confusing to have path read out loud and field name)

Don't include beginning and trailing slashes in path